src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -22,6 +22,71 @@
     public static final String DATE_FORMAT_mm_dd_yyyy_hh_mm_ss_am = "MM/dd/yyyy hh:mm:ss a";
     public static final String DATE_FORMAT_yyyy_MM_dd_T_HH_mm_ss_SSS = "yyyy-MM-dd'T'HH:mm:ss.SSS";
     
+
+    public static final String NON_CONFIDENTIAL_ATTACHMENT_TYPE_LABEL = "Default";
+    
+    // I Want document constants
+    public static final String I_WANT_DOC_ITEM_TAB_ERRORS = "document.item*,newIWantItemLine*";
```

Comment:
This block looks like it just reformatting CU custom constants. Should that be part of this PR?

Reply:
This reformatting block was carried over from the original PR for this upgrade's changes. I don't have a strong preference either way on whether the changes to this file should go forward. (If they do, though, then we would need to rebase this PR if #777 gets merged first.) We can discuss with Daniela tomorrow if needed.

Reply:
@db483 Do you think that the CUKFSConstants code reordering should still be included with this patch? I'm not sure if it's in scope for the patch, given that it's not extending from the base KFSConstants class and our other constants classes haven't been reordered likewise.

Reply:
The reformatting is not needed for the upgrade, it was just to keep the KFSConstants and our local class in similar pattern. I am fine with not including this change if you think that would be better. thank you.

Reply:
Ok, I'll remove this change for now, and we can add it back in for a future user story when needed. Thanks!